boards/x86/minnowboard/minnowboard.dts

```diff
@@ -1,12 +1,7 @@
 /dts-v1/;
 
-#define DT_FLASH_SIZE		__SIZE_K(72)
-
-#if XIP
-	#define DT_SRAM_SIZE		__SIZE_K(128)
-#else
-	#define DT_SRAM_SIZE		__SIZE_K(256)
-#endif
+#define DT_FLASH_SIZE		__SIZE_K(1024)
```

Comment:
Can we please provide the correct values for the minnowboard? The value 1024 was a hack chosen to un-break the build.

Reply:
ram is 2GB and flash is 8MB (We do not really use the flash on this board).